docs/guide/fields.md

```diff
@@ -116,6 +116,51 @@ strawberry_django.field_type_map.update({
 })
 ```
 
+## Including / excluding Django model fields by name
+
```

Comment:
I would add a `warning` here specifying that this should be used with caution because it can expose unwanted data by mistake, specially when using `__all__`

You can check an example of a warning at the `docs/guide/authentication.md` file